generated/nidaqmx/system/physical_channel.py

```diff
@@ -1249,20 +1280,24 @@ def write_to_teds_from_array(
             with cfunc.arglock:
                 if cfunc.argtypes is None:
                     cfunc.argtypes = [
-                        ctypes_byte_str, wrapped_ndpointer(dtype=numpy.uint8,
-                        flags=('C','W')), ctypes.c_uint, ctypes.c_int]
+                        ctypes_byte_str, ctypes_byte_str,
+                        wrapped_ndpointer(dtype=numpy.uint8, flags=('C','W')),
+                        ctypes.c_uint, ctypes.c_int]
 
         error_code = cfunc(
-            self._name, bit_stream, len(bit_stream), basic_teds_options.value)
+            self._name, physical_channel, bit_stream, len(bit_stream),
+            basic_teds_options.value)
         check_for_error(error_code)
 
     def write_to_teds_from_file(
-            self, file_path="",
+            self, physical_channel="", file_path="",
```

Comment:
fyi - this is wrong, too. I think you're fixing all of them, though.

Reply:
Yes, it will fixed when metadata gets updated from Scrapigen

Reply:
Updated Scrapigen `functions.py`.